drafts/source_generators_mvvm_toolkit.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Source Generators in the MVVM Community Toolkit
+domain: software-engineering-corner.hashnode.dev
+tags: dotnet,mvvm,xaml,maui,csharp
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1697099583332/Lv8fEMIab.png?auto=format
+publishAs: NiZarper
+hideFromHashnodeCommunity: false
```

Comment:
Until we release it, we have to add the the property:
`ignorePost: true`
to the post. As soon as we remove this the post will directly be released on the blog.

Reply:
Oh correct, I added it.